content/id/docs/tasks/tools/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Instalasi Alat"
```

Comment:
```suggestion
title: "Instalasi Perkakas"
```

Reply:
How about using "Perkakas"?

https://kbbi.kemdikbud.go.id/entri/perkakas

Reply:
Ah, just checked RSNI and it uses "alat".
I feel like "peralatan" or "perkakas" fits more though. Any of them is ok, at your choice.

Reply:
i'll pick `peralatan`